primitives/src/lib.rs

```diff
@@ -4,6 +4,12 @@ use codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 use sp_runtime::Percent;
 
+#[derive(Encode, Decode, Clone, Copy, Eq, PartialEq, MaxEncodedLen, Debug, TypeInfo)]
+pub enum Percentage {
```

Comment:
Percentage is too vague for this context, it's difficult to tell apart from Percent just by the name.

```suggestion
pub enum OneOrPercent {
```

Reply:
Resolved in [a8c5e0e](https://github.com/InvArch/InvArch-Frames/pull/147/commits/a8c5e0e90b6cb3d4ffd7e21943ba688e4c3cdb6a)